requirements.txt

```diff
@@ -0,0 +1,3 @@
+PyGithub==1.38
+PyJWT==1.6.1
```

Comment:
What are we using `PyJWT` for?

Also, we may want to add our other requirements (e.g. Jinja2) to this file as well.

Reply:
It was a dependency of `PyYaml`, I believe. I added `requirements.txt` prior to realizing that we already listed dependencies in `tox.ini` so this is probably redundant. Want me to just remove it?